src/Cool/Variables/TestVariables.cpp

```diff
@@ -1,14 +1,13 @@
-#include "Cool/StrongTypes/TextureWrapper.h"
 #if DEBUG
 
+#include "TestVariables.h"
 #include <Cool/Variables/Variables.h>
 #include <imgui/imgui.h>
 #include <vector>
 #include "Cool/Camera/Camera.h"
 #include "Cool/StrongTypes/ColorAndAlpha.h"
 #include "Cool/StrongTypes/Point2D.h"
-#include "TestVariables.h"
-
+#include "Cool/StrongTypes/TextureWrapper.h"
 
```

Comment:
No need to include MathExpression here? If true, maybe we can remove the others includes ?